module/VuFind/src/VuFind/RecordDriver/SolrOverdrive.php

```diff
@@ -272,11 +271,11 @@ public function getOverdriveAccess()
      *
      * Returns whether the current user is logged in
      *
-     * @return object|bool User if logged in, false if not.
+     * @return bool User if logged in, false if not.
```

Comment:
This comment is no longer accurate with the changed return type -- I'd just drop the comment entirely since the method is now self-explanatory, and reduce the line to simply `@return bool`